src/Azure/AzureAD/Authentication.AzureAD.UI/src/AzureADAuthenticationBuilderExtensions.cs

```diff
@@ -28,6 +29,7 @@ public static class AzureADAuthenticationBuilderExtensions
         /// <see cref="AzureADOptions"/>.
         /// </param>
         /// <returns>The <see cref="AuthenticationBuilder"/>.</returns>
+        [Obsolete("This is obsolete and will be removed in a future version. Use AddMicrosoftWebApiAuthentication from Microsoft.Identity.Web instead. See https://aka.ms/ms-identity-web.")]
```

Comment:
This is a pattern we use to obsolete extension functions on a static class. I've never verified but I assume that it's because we want any use of the static class to elicit the warning as well as uses of the extension methods (which doesn't use the static class in any way) to elicit the same warning. Not sure if it's required though but it can't hurt. Also, the extension methods have a slightly different message than the class.